cypress/integration/core/viewer-grid.spec.ts

```diff
@@ -102,7 +102,6 @@ describe("Viewer Grid", () => {
       .find("div[slot = 'overlay'] a[href]")
       .eq(1)
       .should("be.focused")
-      .realPress("Tab")
```

Comment:
Deze Tab is in de nieuwe situatie met huidige (compactere) vulling van de `overlay` niet meer nodig

Reply:
Wat bedoel je met "compactere vulling" ?

Reply:
Voorheen stond er in de `overlay panel` een flinke lap tekst, met een aantal hyperlinks en een afbeelding met en vergrootknop. Daar moest in de test overheen getabt worden om weer terug bij het sluitkruis te kunnen komen.

Nu staan er in dezelfde panel nog maar 2 hyperlinks, dus deze extra tab is niet meer nodig om bij het sluitkruis te komen.